docs/source/models/intervention_models/neonatal/cpap_intervention.rst

```diff
@@ -126,13 +136,20 @@ Where,
     - Relative risk of CPAP intervention on RDS mortality 
     - 0.53 (95% CI 0.34-0.83). Uncertaintly interval implemented as parameter uncertainty following a lognormal distribution
     - `2020 Cochrane review <https://pmc.ncbi.nlm.nih.gov/articles/PMC8094155/>`_. Note that this effect was measured for all cause mortality.
+  * - :math:`\text{RR}_\text{ACS}`
+    - Relative risk of ACS intervention on RDS mortality 
+    - Refer to :ref:`ACS intervention page <acs_intervention>` for this effect size.
+    - Only to be included in PAF calculation if simulant is within the gestational age range that is eligible for ACS (26-33 weeks).
 
 .. _cpap_calibration:
 
 Calibration Strategy
 --------------------
 
-The following decision tree shows all of the paths from delivery facility choice to CPAP availability.  Distinct paths in the tree correspond to disjoint events, which we can sum over to find the population probability of RDS mortality.  The goal here is to find internally consistent probabilities of RDS mortality for the subpopulations with and without access to CPAP, so that the baseline scenario can track who has access to CPAP and still match the baseline RDS mortality rate.
+The following decision tree shows all of the paths from delivery facility choice to CPAP availability.  Distinct paths in the tree correspond to 
+disjoint events, which we can sum over to find the population probability of RDS mortality.  The goal here is to find internally consistent probabilities 
+of RDS mortality for the subpopulations with and without access to CPAP and ACS, so that the baseline scenario can track who has access to CPAP and still match 
+the baseline RDS mortality rate.
```

Comment:
Just adding line breaks, right? I usually slightly prefer to add them between sentences but maybe that's just me